### DISCUSSION TITLE
Set filepath in player.begin()

### DISCUSSION THREAD

**Discussion by Gaai** [pschatzmann/arduino-audio-tools#801] [Q&A] [answered]
A bit silly of me, but I thought it intuitive to set the path before player.begin(). Trying different ways:
- SD.open("filename.wav")
- player.setPath("filename.wav") 
- source.selectStream("filename.wav")

Which obviously had no effect before player.begin() because that does a setIndex(0) by default. 
Anyway, I thought I would post this message for others stumbling around in the dark. Also it might be worth to consider adding filename/path as an optional parameter to player.begin? 
Do let me know if I missed something. 

Thanks for the continuous effort to create and update this awesome library!

**Comment by pschatzmann** [comment, marked as answer]
I thought most of the examples show how to use this: e.g. https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-audiokit/player-sd-audiokit/player-sd-audiokit.ino

Now I realize, that this is true for the AudioKit examples only...

**Reply by Gaai** [reply]
That one does indeed! Hope others pay more attention, or read this message ;)
